src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -151,9 +158,26 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
   }
 
   handleLogsError(data: PanelData) {
+    const errorResponse = data.errors?.length ? data.errors[0].message : data.error?.message;
```

Comment:
I think we should log the error response using our logger.

Reply:
Great idea.